docs/includes/swagger/cds_full.json

```diff
@@ -2037,26 +2037,26 @@
         "feeType" : {
           "type" : "string",
           "description" : "The type of fee",
-          "enum" : [ "PERIODIC", "TRANSACTION", "WITHDRAWAL", "DEPOSIT", "PAYMENT", "PURCHASE", "EVENT", "UPFRONT", "EXIT" ]
+          "enum" : [ "PERIODIC", "TRANSACTION", "WITHDRAWAL", "DEPOSIT", "PAYMENT", "PURCHASE", "EVENT", "UPFRONT", "EXIT", "VARIABLE" ]
```

Comment:
Updated inline for existing payloads with enumeration v2. Opened ticket related to payload version change: https://github.com/ConsumerDataStandardsAustralia/standards-maintenance/issues/310